CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+## 2018-01-26 - Version 4.2.2
+
+### Summary
+
+This is a minor release with bug fixes, and one new property added to manage connection limits for IIS Sites.
+
+### Added
+
+- Setting the ConnectionTimeout for limits to drop inactive connections ([MODULES-6144](https://tickets.puppetlabs.com/browse/MODULES-6144))
```

Comment:
This should be site limits and include connection timeout, maxbandwidth, and max connections.